Cargo.toml

```diff
@@ -120,6 +121,7 @@ axum-test = { version = "17.0.1", optional = true }
 tree-fs = { version = "0.3", optional = true }
 
 chrono = { workspace = true }
+time = "0.3.41"
```

Comment:
Why not use chrono?

Reply:
The `axum_csrf` crate uses another [`Duration`](https://docs.rs/time/latest/src/time/duration.rs.html#43-50) struct that distincts from the one in the `chrono` crate. I think this is because the `Duration` struct in the `time` crate also allows [negative durations](https://docs.rs/time/latest/time/struct.Duration.html).

This is why I had to use this crate…

Reply:
Hey @floscodes,

I still feel that adding this crate, especially alongside another dependency isn't worth the tradeoff.
Implementing it manually in the `after_route` (as I initially suggested) would provide more value to users, since it gives them full control over configuration and crate versions. We could even publish a short guide on how to do it.

Given that, I think it's better to close this PR.

Reply:
Ok. So maybe there could be a short guide on the docs side...